src/common/src/types/decimal.rs

```diff
@@ -30,7 +31,7 @@ use crate::error::Result as RwResult;
 use crate::types::ordered_float::OrderedFloat;
 use crate::types::Decimal::Normalized;
 
-#[derive(Debug, Copy, parse_display::Display, Clone, PartialEq, Hash, Eq, Ord, PartialOrd)]
+#[derive(Debug, Copy, parse_display::Display, Clone, PartialEq, Hash, Eq)]
```

Comment:
It will work as well. But implementing the `ord` trait is more clean and efficient way of handling such kind of things, imo. In case, anything changes in future, one only needs to change these explicit functions.